include/fastdds/dds/xtypes/dynamic_types/DynamicType.hpp

```diff
@@ -60,6 +60,7 @@ class DynamicType : public std::enable_shared_from_this<DynamicType>
     /*!
      * Returns the member that corresponds to the specified name.
      * @param[inout] member @ref DynamicTypeMember reference used to return the reference to the member.
+     * @param[in] name the name of the member.
```

Comment:
```suggestion
     * @param[in] name Member name of the member being queried.
```

Reply:
Fixed in [8020bf2](https://github.com/eProsima/Fast-DDS/pull/4240/commits/8020bf2136b0ed793a5e093b26fc59595dbd7b26)